altair/examples/dendrogram.py

```diff
@@ -8,7 +8,12 @@
 
 import pandas as pd
 import altair as alt
+import numpy as np
 
+# the variable `den` shown below is an exemplary output of `scipy.cluster.hierarchy.dendrogram`, where
+# the dendrogram itself is truncated such that no more than 3 levels of the dendrogram tree are shown.
+# Please refer to https://scikit-learn.org/stable/auto_examples/cluster/plot_agglomerative_dendrogram.html
```

Comment:
Should this link to https://docs.scipy.org/doc/scipy/reference/generated/scipy.cluster.hierarchy.dendrogram.html#scipy.cluster.hierarchy.dendrogram instead? It makes sense that the link in the text is to the sklearn example but just as a reference to the dendrogram I think linking the docs would be more suitable (unless there is some non-standard use of heriarchy.dendrogram in the sklearn example that is necessary for this to work

Reply:
Agreed. The scikit-learn example link was already provided in the abstract below the title anyway.